monitoring/metrics.py

```diff
@@ -0,0 +1,22 @@
+from google.api import label_pb2 as ga_label
```

Comment:
Are we using this file somewhere? I don't see it mentioned in the README or called from anywhere